lms/templates/preview_menu.html

```diff
@@ -77,8 +77,7 @@
                 </div>
             % endif
             % if user.is_staff and mfe_link:
-            <div style="text-align: right;float:right;padding-bottom:5px">
-                <p style="color: white; display: inline-block; margin-right: 10px;">We're working on a new learning sequence experience</p>
+            <div style="flex-grow: 1; text-align: right;">
                 <a class="btn btn-primary" style="border: solid 1px white;" href="${mfe_link}">View this unit in the new experience</a>
```

Comment:
This isn't part of your work, but could you please add i18n translation here?

Reply:
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#mako-template-files